src/navs/index.js

```diff
@@ -1870,6 +1870,12 @@ export const docsMenu = {
                     icon: 'Bolt',
                     color: 'red',
                 },
+                {
+                    name: 'Data management',
+                    url: '/docs/product-analytics/data-management',
```

Comment:
Could we just link to `/docs/data` here? And add more links to data management in the various product analytics pages? I think another page to maintain that has duplicate content isn't ideal.

Reply:
Hmm interesting

Pros of your suggestion:
- Less duplication

Cons:
- Might feel weird for the user to be navigated somewhere else in the docs unexpectedly
- Current "[data management overview](https://posthog.com/docs/data)" page isn't that great, so it will be confusing to the user

I think if I update the data management overview page to provide more context (and maybe move/change the "definitions" stuff which is confusing), then it should be fine.

I'll give it some more thought, but I think I'll go with your suggestion!